documentation/src/docs/beginner_dev_guide.md

```diff
@@ -0,0 +1,202 @@
+<!--
+Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+# Pravega - Creating Your First Application
+
+Learn how to create a Hello World Pravega app. This guide covers:
+* Starting Pravega standalone
+* Creating a Pravega Stream.
+* Creating an Event Writer and write events into a Pravega Stream.
+* Create an Event Reader and read events from the Pravega Stream.
+
+ 
+# 1. Prerequisites
+To complete this guide, you need:
+* less than 15 minutes
+* an IDE
+* JDK 11+ installed with JAVA_HOME configured appropriately
+* <details>
+  <summary>Gradle 6.5.1+</summary>
+  Installation : https://gradle.org/install/
+  
+  !! Verify Gradle is using the Java you expect. You can verify which JDK Gradle uses by running `gradle --version`.!!
+</details>
+
+# 2. Goal
+In this guide, we will create a straightforward application that creates a Stream on Pravega and writes an event into the Stream and reads back from it.
+We recommend that you follow the instructions from [Bootstrapping project](#4-Bootstrapping-the-Project) onwards to create the application step by step.
+However, you can go straight to the completed example at [Pravega-samples-repo](https://github.com/pravega/pravega-samples).
+
+<details>
+<summary>Command to clone the pravega-samples repo</summary>
+<p>
+
+```java
+git clone https://github.com/pravega/pravega-samples.git
+
+```
+
+</p>
+</details>  
+
+
+The solution is located in the pravega-client-examples [directory]( https://github.com/pravega/pravega-samples/tree/master/pravega-client-examples/src/main/java/io/pravega/example/gettingstarted ).
+
+# 3. Starting Pravega Standalone.
+In standalone mode, the Pravega server is accessible from clients through the `localhost` interface only. Controller REST APIs, however, are accessible from remote hosts/machines.
+You can launch a standalone mode server using either of the following options:
+
+1. From source code
+
+<details>
+<summary>Commands to start standalone from source code</summary>
+<p>
+
+Checkout the source code:
+```java
+$ git clone https://github.com/pravega/pravega.git
+$ cd pravega
+```
+Build the Pravega standalone mode distribution:
+
+```java
+./gradlew startStandalone
+```
+
+</p>
+</details>
+
+2. From installation package
+
+<details>
+<summary>Commands to start standalone from installation package</summary>
+<p>
+Download the Pravega release from the [GitHub Releases](https://github.com/pravega/pravega/releases).
```

Comment:
The hyperlink for the text `GitHub Releases` is not showing up correctly

Reply:
fixed.